core/modules/qdisp/QueryRequest.cc

```diff
@@ -58,7 +58,7 @@ inline void unprovisionSession(XrdSsiSession* session) {
 }
 
 ////////////////////////////////////////////////////////////////////////
-// QueryRequest::Canceller
+// QueryRequest::Canceler
 ////////////////////////////////////////////////////////////////////////
 class QueryRequest::Canceller : public util::VoidCallable<void> {
```

Comment:
Yup, just consistent throughout.  I'm good with two ells.  (John sez he tried to put this back but it was more merge lossage.)